site/_docs/history.md

```diff
@@ -56,19 +146,297 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6269">CALCITE-6269</a>]
+  Fix missing/broken BigQuery date-time format elements
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6231">CALCITE-6231</a>]
+  JDBC adapter generates `UNNEST` when it should generate `UNNEST ... WITH ORDINALITY`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6208">CALCITE-6208</a>]
+  Update `JSON_VALUE` return type inference to make explicit array return types be nullable with nullable elements
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6199">CALCITE-6199</a>]
+  Trim unused fields for `SNAPSHOT` and `SAMPLE` if table has `VIRTUAL` column
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6063">CALCITE-6063</a>]
+  If `ARRAY` subquery has `ORDER BY` (without `LIMIT`), rows are not sorted
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6032">CALCITE-6032</a>]
+  Multilevel correlated query is failing in `RelDecorrelator` code path
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6254">CALCITE-6254</a>]
+  Support table function calls in `FROM` clause without `TABLE()` wrapper
 
 #### Build and test suite
 {: #build-1-37-0}
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6103">CALCITE-6103</a>]
+ Use eclipse-temurin image to build and publish javadocs for the website
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6131">CALCITE-6131</a>]
+ There are duplicate sample tests in `SqlTypeUtilTest`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6105">CALCITE-6105</a>]
+ Documentation does not specify the behavior of `SPLIT` function for empty string arguments
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6125">CALCITE-6125</a>]
+ Automate generation of contributor names in release notes by adding a git mailmap file
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6165">CALCITE-6165</a>]
+ Add `DATE_ADD` test and `DATE_DIFF` test on `SqlOperatorTest`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6184">CALCITE-6184</a>]
+ Add `checkNullTest` on `SqlOperatorTest`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6187">CALCITE-6187</a>]
+ Linter should disallow tags such as `[MINOR]` in commit messages
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6273">CALCITE-6273</a>]
+ Add sqrt negative test in `SqlOperatorTest`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6189">CALCITE-6189</a>]
+ Improve `FormatElementEnumTest`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6234">CALCITE-6234</a>]
+ Add tests on `SqlOperatorTest` for `to_char` function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6172">CALCITE-6172</a>]
+ Allow aliased operators to re-use existing tests
 
 #### Web site and documentation
 {: #site-1-37-0}
--->
+Site: Switch PMC Chair to Benchao Li
+Site: Troubleshooting/Website publishing improvements in release guide
+Site: Remove committer by request process
```

Comment:
These three lines should start with `*` ?

Reply:
yep, thanks
Also double checked with prev release notes